var/spack/repos/builtin/packages/abyss/package.py

```diff
@@ -34,9 +34,11 @@ class Abyss(AutotoolsPackage):
     url      = "http://www.bcgsc.ca/platform/bioinfo/software/abyss/releases/2.0.2/abyss-2.0.2.tar.gz"
 
     version('2.0.2', '1623f55ad7f4586e80f6e74b1f27c798')
+    version('1.5.2.', '10d6d72d1a915e618d41a5cbbcf2364c', url='https://github.com/bcgsc/abyss/releases/download/1.5.2/abyss-1.5.2.tar.gz')
```

Comment:
It would be good to get both releases from the same source if possible. Also, you accidentally added a period at the end of the version number.